blockchain/chain_sync/src/tipset_syncer.rs

```diff
@@ -1109,7 +1139,7 @@ async fn sync_messages_check_state<DB: BlockStore + Send + Sync + 'static, C: Co
 async fn validate_tipset<DB: BlockStore + Send + Sync + 'static, C: Consensus>(
     consensus: Arc<C>,
     state_manager: Arc<StateManager<DB>>,
-    chainstore: Arc<ChainStore<DB>>,
+    chainstore: &Arc<ChainStore<DB>>,
     bad_block_cache: Arc<BadBlockCache>,
```

Comment:
This can just be a reference:
```suggestion
    bad_block_cache: &BadBlockCache,
```